src/lib.rs

```diff
@@ -28,12 +28,18 @@ The following code recursively iterates over the directory given and prints
 the path for each entry:
 
 ```rust,no_run
-use walkdir::WalkDir;
+use walkdir::{Error, WalkDir};
 
+# fn try_main() -> Result<(), Error> {
```

Comment:
Again, I was following this guideline and they use try_main() there:
https://github.com/brson/rust-api-guidelines#examples-use--not-try-not-unwrap-c-question-mark